MapboxNavigation/Geometry.swift

```diff
@@ -82,12 +82,12 @@ extension CLLocationCoordinate2D {
     }
     
     /// Returns the direction from the receiver to the given coordinate.
-    func direction(to coordinate: CLLocationCoordinate2D) -> CLLocationDirection {
+    public func direction(to coordinate: CLLocationCoordinate2D) -> CLLocationDirection {
```

Comment:
Is it necessary to make these methods public, or can we keep access limited to this module until we spin off an independent Geometry.swift as part of mapbox-swift-core?